ui/imports/shared/popups/ProfilePopup.qml

```diff
@@ -47,12 +47,14 @@ StatusModal {
     signal contactBlocked(publicKey: string)
     signal contactAdded(publicKey: string)
 
-    function openPopup(publicKey, openNicknamePopup) {
+    function openPopup(publicKey, openNicknamePopup) {        
+        isCurrentUser = popup.profileStore.pubkey === publicKey
+
         // All this should be improved more, but for now we leave it like this.
         let contactDetails = Utils.getContactDetailsAsJson(publicKey)
         userPublicKey = publicKey
-        userDisplayName = contactDetails.displayName
-        userName = contactDetails.alias
+        userDisplayName = isCurrentUser ? contactDetails.name: contactDetails.displayName
```

Comment:
@Khushboo-dev-cpp I am not sure that we should taking into consideration `isCurrentUser` variable. Using `displayName` contains some logic behind, but with this change will this work well for user who has ens name set?

Calling `Utils.getContactDetailsAsJson` you're calling [this proc](https://github.com/status-im/status-desktop/blob/a00b182b8b4e920d2b1d45c18e61534b02ac1251/src/app/modules/main/module.nim#L621-L642) and if you follow calls from it you will see that display name in has some [logic](https://github.com/status-im/status-desktop/blob/a00b182b8b4e920d2b1d45c18e61534b02ac1251/src/app_service/service/contacts/dto/contacts.nim#L87-L95) behind and in case of [logged in user](https://github.com/status-im/status-desktop/blob/a00b182b8b4e920d2b1d45c18e61534b02ac1251/src/app/global/user_profile.nim#L128-L137), so I think that there is some discrepancy on the backend side, between logic used for contact dto and one used for logged in user.

If you don't mind, please check that, cause fixing it this way is a local change, and the issue may appear on other place where we're using `Utils.getContactDetailsAsJson`, but all contacts' display names must be the same across the app.

Reply:
> @Khushboo-dev-cpp I am not sure that we should taking into consideration `isCurrentUser` variable. Using `displayName` contains some logic behind, but with this change will this work well for user who has ens name set?
> 
> Calling `Utils.getContactDetailsAsJson` you're calling [this proc](https://github.com/status-im/status-desktop/blob/a00b182b8b4e920d2b1d45c18e61534b02ac1251/src/app/modules/main/module.nim#L621-L642) and if you follow calls from it you will see that display name in has some [logic](https://github.com/status-im/status-desktop/blob/a00b182b8b4e920d2b1d45c18e61534b02ac1251/src/app_service/service/contacts/dto/contacts.nim#L87-L95) behind and in case of [logged in user](https://github.com/status-im/status-desktop/blob/a00b182b8b4e920d2b1d45c18e61534b02ac1251/src/app/global/user_profile.nim#L128-L137), so I think that there is some discrepancy on the backend side, between logic used for contact dto and one used for logged in user.
> 
> If you don't mind, please check that, cause fixing it this way is a local change, and the issue may appear on other place where we're using `Utils.getContactDetailsAsJson`, but all contacts' display names must be the same across the app.

Hey Sale, Good point! I was wondering the same when implementing it. The thing is, what should happen when the user has ENS name? is it visible on the profile screen ?